src/Card.js

```diff
@@ -3,13 +3,33 @@ import CardDescription from './Card-Description';
 import CardStyle from './Card.css';
 
 export default class Card extends Component {
-    render() {
+  constructor() {
+    super();
+    this.state = {showInfo: false}
+  }
+
+  toggleInfo = (event) => {
+    this.setState({showInfo: !this.state.showInfo});
+    event.target.previousSibling.classList.toggle('expanded-description');
+    event.target.previousSibling.previousSibling.classList.toggle('expanded-img');
+    event.target.parentElement.classList.toggle('expanded-card');
```

Comment:
You won't see classes being added/removed from elements like this. (By accessing the DOM elements near `event.target`.) Instead, you should change some logic in your render method to allow for conditional classes. Take a look at the first answer listed here: https://stackoverflow.com/questions/30533171/react-js-conditionally-applying-class-attributes